### PR TITLE
Update build-instructions.md: clarify where to use -DTARGET_i386=OFF flag.

### PR DESCRIPTION
**Pull request by wyattscarpenter**

### CODE REVIEW

File: src/build-instructions.md [hunk 1]
```diff
@@ -171,6 +171,7 @@ sudo make install
 Darling normally builds both 32-bit and 64-bit versions of all libraries, to enable 32-bit programs to run under Darling.
 However, this means Darling also requires 32-bit version of certain native libraries. If you can't setup a multilib environment or you just
 want to build only the 64-bit components, use `-DTARGET_i386=OFF` during configuration to disable building the 32-bit components.
```
Comment (CuriousTommy) [inline src/build-instructions.md:173]: I would word it like this instead
```suggestion
want to build only the 64-bit components, append `-DTARGET_i386=OFF` when you run `cmake` to disable building the 32-bit components.
```

**Review by CuriousTommy — CHANGES_REQUESTED** [review]
The `during configuration` part is meant to imply that you should use `cmake`, but I also understand that this might not be obvious for someone who isn't familiar with `make`/`cmake`.

For consistency, I would also update the following to make it more clear that you should use `cmake`:
https://github.com/darlinghq/darling-docs/blob/99f2a324b82fb5191f657f78d99b6f67e508ee98/src/build-instructions.md?plain=1#L181
https://github.com/darlinghq/darling-docs/blob/99f2a324b82fb5191f657f78d99b6f67e508ee98/src/build-instructions.md?plain=1#L196
https://github.com/darlinghq/darling-docs/blob/99f2a324b82fb5191f657f78d99b6f67e508ee98/src/build-instructions.md?plain=1#L200
https://github.com/darlinghq/darling-docs/blob/99f2a324b82fb5191f657f78d99b6f67e508ee98/src/build-instructions.md?plain=1#L224

**Comment by wyattscarpenter** [comment]
I incorporated your change and took at stab at the other changes, which I agree would be good to clarify.

Notably, I don't actually know how COMPONENTS works, because I don't know cmake, so I guessed.

**Review by CuriousTommy — APPROVED** [review]
LGTM

**Comment by CuriousTommy** [comment]
Thank you for your contribution!

**Pull request merged**